tests/mountingOptions/global.stubs.spec.ts

```diff
@@ -679,16 +679,16 @@ describe('mounting options: stubs', () => {
   })
 
   describe('stub async component', () => {
-    const AsyncComponent = defineAsyncComponent(async () => ({
-      name: 'AsyncComponent',
-      template: '<span>AsyncComponent</span>'
-    }))
-
     const AsyncComponentWithoutName = defineAsyncComponent(async () => ({
       template: '<span>AsyncComponent</span>'
     }))
 
     it('stubs async component with name', async () => {
+      const AsyncComponent = defineAsyncComponent(async () => ({
```

Comment:
We need to move component definition inside test, because otherwise for second test `__asyncResolved` will be already defined from previous test